Modules/Loadable/Sequences/MRML/vtkMRMLSequenceBrowserNode.cxx

```diff
@@ -49,13 +49,17 @@
 #  define SNPRINTF snprintf
 #endif
 
-// First reference is the master sequence node, subsequent references are the synchronized sequence nodes
-static const char* SEQUENCE_NODE_REFERENCE_ROLE_BASE = "sequenceNodeRef"; // Old: rootNodeRef
-// TODO: Change this to "proxyNodeRef", but need to maintain backwards-compatibility with "dataNodeRef"
-static const char* PROXY_NODE_REFERENCE_ROLE_BASE = "dataNodeRef";
+namespace
+{
+  // First reference is the master sequence node, subsequent references are the synchronized sequence nodes
+  const char* SEQUENCE_NODE_REFERENCE_ROLE_BASE = "sequenceNodeRef"; // Old: rootNodeRef
+  // TODO: Change this to "proxyNodeRef", but need to maintain backwards-compatibility with "dataNodeRef"
```

Comment:
Should an issue be created ? Or should the comment be changed to:

```
  // Ideally this should be changed to "proxyNodeRef" but we need to maintain it
  // for backwards-compatibility with "dataNodeRef"
```

Reply:
Good point, I'll use the text you've suggested.